sdk-tests/src/test/java/io/dapr/it/pubsub/http/PubSubIT.java

```diff
@@ -402,6 +404,58 @@ public void testPubSubTTLMetadata() throws Exception {
     daprRun.stop();
   }
 
+  @Test
+  public void testlongValues() throws Exception {
+    final DaprRun daprRun = closeLater(startDaprApp(
+        this.getClass().getSimpleName(),
+        SubscriberService.SUCCESS_MESSAGE,
+        SubscriberService.class,
+        true,
+        60000));
+    // At this point, it is guaranteed that the service above is running and all ports being listened to.
+    if (this.useGrpc) {
+      daprRun.switchToGRPC();
+    } else {
+      daprRun.switchToHTTP();
+    }
+
+    ConvertToLong value = new ConvertToLong();
+    value.setVal(590518626939830271L);
+    try (DaprClient client = new DaprClientBuilder().build()) {
+      for (int i = 0; i < NUM_MESSAGES; i++) {
+        //Publishing messages
+        client.publishEvent(
+            PUBSUB_NAME,
+            LONG_TOPIC_NAME,
+            value,
+            Collections.singletonMap(Metadata.TTL_IN_SECONDS, "1")).block();
+
+        try {
+          Thread.sleep((long) (1000 * Math.random()));
+        } catch (InterruptedException e) {
+          e.printStackTrace();
+          Thread.currentThread().interrupt();
+          return;
+        }
+      }
+    }
+
+    try (DaprClient client = new DaprClientBuilder().build()) {
+      callWithRetry(() -> {
+        System.out.println("Checking results for topic " + LONG_TOPIC_NAME);
+        final List<CloudEvent> messages = client.invokeMethod(
+            daprRun.getAppName(),
+            "messages/testinglongvalues",
+            null,
+            HttpExtension.GET, CLOUD_EVENT_LIST_TYPE_REF).block();
+        for (int i = 0; i < NUM_MESSAGES; i++) { 
+          System.out.println("The long value received!!! " +messages.get(i).getData().value);
+          assertEquals(590518626939830271L, messages.get(i).getData().value);
+        }
+      }, 2000);
+    }
```

Comment:
Add assert comparing two set values 
```java
Assert.assertEquals(expected, actual);
```
Both `expected` and `actual` are of type `Set<ConvertToLong`